projects/js-packages/charts/src/components/pie-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,406 @@
+	/>` }
+/>
+
+#### Rounded Corners
+
+Apply `cornerScale` for rounded segment corners:
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		data={ data }
+		cornerScale={ 0.1 } // 10% corner rounding
+	/>` }
+/>
+
+#### Thickness Control
+
+Adjust `thickness` to create various chart styles:
+
+<Source
+	language="jsx"
+	code={ `// Full pie chart (default)
+	<PieChart thickness={ 1 } data={ data } />
+
+	// Pie chart with slight inner space
+	<PieChart thickness={ 0.9 } data={ data } />` }
+/>
+
+### Custom Colors
+
+Override theme colors by providing color values in data:
+
+<Source
+	language="jsx"
+	code={ `const dataWithColors = [
+		{
+			label: 'MacOS',
+			value: 30000,
+			percentage: 23,
+			color: '#007cba' // Custom blue
+		},
+		{
+			label: 'Linux',
+			value: 22000,
+			percentage: 17,
+			color: '#f56565' // Custom red
+		},
+		{
+			label: 'Windows',
+			value: 80000,
+			percentage: 60,
+			color: '#48bb78' // Custom green
+		},
+	];
+
+	<PieChart data={ dataWithColors } />` }
+/>
+
+### Theme Integration
+
+Pie Charts automatically integrate with the chart theme system, inheriting colors, typography, and styling from the active theme. Custom themes can be applied using the `ThemeProvider`:
+
+<Source
+	language="jsx"
+	code={ `import { ThemeProvider, jetpackTheme } from '@automattic/charts';
+
+	<ThemeProvider theme={ jetpackTheme }>
+		<PieChart data={ data } />
+	</ThemeProvider>` }
+/>
+
+## Advanced Features
+
+### Responsive Behavior
+
+The Pie Chart component supports responsive sizing by omitting the `size` prop:
+
+<Canvas of={ PieChartStories.Responsiveness } />
+
+<Source
+	language="jsx"
+	code={ `// Responsive - fills parent container
+	<PieChart
+		data={ data }
+		maxWidth={ 600 }
+		aspectRatio={ 1 }
+	/>
+
+	// Fixed size
+	<PieChart
+		size={ 400 }
+		data={ data }
+	/>` }
+/>
+
+### Data Validation and Error Handling
+
+The component includes built-in data validation with helpful error states:
+
+<Canvas of={ PieChartStories.ErrorStates } />
+
+The chart validates:
+- **Empty data**: Shows appropriate message when no data is provided
+- **Percentage totals**: Ensures percentages sum to 100%
+- **Negative values**: Prevents invalid negative values
+- **Single data points**: Handles edge cases gracefully
+
+## Accessibility
+
+### Screen Reader Support
+
+- **ARIA labels**: Each chart segment includes descriptive `aria-label` attributes
+- **Semantic structure**: Proper heading hierarchy and landmark regions
+- **Data table alternative**: Consider providing a data table alternative for complex charts
+- **Tooltip announcements**: Screen readers announce tooltip content when segments are focused
+
+### Color Accessibility
+
+- **Sufficient contrast**: Default theme colors meet WCAG AA contrast requirements
+- **Colorblind friendly**: Default palette works well for common color vision deficiencies
+- **Pattern alternatives**: Consider using patterns or textures for critical distinctions
```

Comment:
These need to be checked, I think there's some hallucination. For example I can't see Aria labels

Reply:
Removed